addons/hr_expense/models/hr_expense.py

```diff
@@ -8,7 +8,7 @@
 
 from odoo import api, fields, Command, models, _
 from odoo.exceptions import RedirectWarning, UserError, ValidationError
-from odoo.tools import clean_context, email_normalize, float_repr, float_round, is_html_empty
+from odoo.tools import clean_context, email_normalize, float_repr, float_round, is_html_empty, format_date
```

Comment:
```suggestion
from odoo.tools import clean_context, email_normalize, float_repr, float_round, format_date, is_html_empty
```
Alphabetical Order :eyes: